### ISSUE TITLE
Alan Wake II - HDR not working

### ISSUE THREAD

**Issue by DomiStyle** [HansKristian-Work/vkd3d-proton#1755]
When started through gamescope with HDR enabled, HDR is greyed out in the Brightness calibration menu in Alan Wake 2.

![image](https://github.com/HansKristian-Work/vkd3d-proton/assets/975131/8931a0d0-b0f4-46a4-a436-40c935186b91)

### Software information
Alan Wake 2, gamescope is started from a TTY via this script:
```
#!/bin/sh

export STEAM_GAMESCOPE_HAS_TEARING_SUPPORT=1
export STEAM_GAMESCOPE_TEARING_SUPPORTED=1
export STEAM_ENABLE_VOLUME_HANDLER=1
export STEAM_DISABLE_AUDIO_DEVICE_SWITCHING=1

export ENABLE_GAMESCOPE_WSI=1
export DXVK_HDR=1

gamescope -w 3440 -h 1440 -W 3440 -H 1440 --immediate-flips --nested-refresh 165 --prefer-output DP-1 --steam --fullscreen --adaptive-sync --hdr-enabled --hdr-itm-enable --sdr-gamut-wideness 1 -- steam -gamepadui
```

### System information
- GPU: AMD 6900 XT
- Driver: Mesa 23.2.1 (happens on mesa-git as well)
- Wine version: Proton Experimental 8.0-2023101
- VKD3D-Proton version: 2.10.0

### Log files
[steam-14472711212434980864.log](https://github.com/HansKristian-Work/vkd3d-proton/files/13195693/steam-14472711212434980864.log)

Log file shows game start, brightness menu and game exit.

**Comment by Joshua-Ashton** [comment]
I was not able to get the game to launch at all with Proton unfortunately =(

**Comment by Joshua-Ashton** [comment]
I am still not able to test the game, but are you able to test with https://github.com/doitsujin/dxvk/pull/3703 ?

This gets Control HDR working.

**Comment by DomiStyle** [comment]
Still greyed out unfortunately, Control (with mod) worked fine though.

**Comment by Filoppi** [comment]
Please contact me if you need help with fixing this, I worked on the HDR implementation on the game so I might be able to figure out what breaks here.
https://learn.microsoft.com/en-us/uwp/api/windows.graphics.display.advancedcolorinfo?view=winrt-22621 it could be that this is not implemented?

**Comment by Joshua-Ashton** [comment]
Thanks for the heads up, really appreciate you reaching out.

Are you actually using the WinRT you linked there or just the stuff in `DISPLAYCONFIG_GET_ADVANCED_COLOR_INFO`?

**Comment by Filoppi** [comment]
Yes, I think I use https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-displayconfiggetdeviceinfo and https://microsoft.github.io/windows-docs-rs/doc/windows/Win32/Devices/Display/struct.DISPLAYCONFIG_GET_ADVANCED_COLOR_INFO.html and also the set methods, that's how we enable HDR.

**Comment by DomiStyle** [comment]
@Blisto91 HDR toggle works fine now!

Display peak brightness is detected as 1499 nits for some reason, even though the display is running in HDR peak 1000 mode. Not sure where it sources that info.

![IMG_20231030_191832](https://github.com/HansKristian-Work/vkd3d-proton/assets/975131/537c0674-4d50-48d0-a7b9-7be019193879)


HDR is definitely enabled, those flashlights are bright! Blacks are washed out though. Might be a calibration issue on my end, so will play around with it a little.

**Comment by Filoppi** [comment]
1500 (or 1499) is the default value Windows assigns as peak brightness if it didn't find any value in the monitor EDID and f the users didn't calibrate.

**Comment by DomiStyle** [comment]
Closing this since it seems fixed.

I'm halfway through the game and HDR looks beautiful, thanks for the quick fix!
